modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/editor/Editor.js

```diff
@@ -13,15 +13,54 @@
  */
 
 import CKEditor from 'ckeditor4-react';
-import React from 'react';
+import React, {useEffect, useState} from 'react';
 
-const BASEPATH = '/o/frontend-editor-ckeditor-web/ckeditor/';
+const CKEDITOR_BASEPATH = '/o/frontend-editor-ckeditor-web/ckeditor/';
+const CKEDITOR_PATH = `${CKEDITOR_BASEPATH}ckeditor.js`;
 
 const Editor = React.forwardRef((props, ref) => {
-	return <CKEditor ref={ref} {...props} />;
+	const [initialized, setInitialized] = useState(!!window.CKEDITOR);
+
+	useEffect(() => {
+		let script;
+
+		const destroyGlobalCkEditor = () => {
+			if (
+				window.CKEDITOR &&
+				Object.keys(window.CKEDITOR.instances).length === 0
+			) {
+				window.CKEDITOR = undefined;
```

Comment:
Instead of assigment, you could actually `delete` it:

```js
window.CKEDITOR = undefined
'CKEDITOR' in window // true
delete window.CKEDITOR
'CKEDITOR' in window // false
```

Which may explain why the `in` check wasn't working before.

Reply:
That's it! ~~Javascript~~ Greg is awesome! A simple thing, when you stop to think :slightly_smiling_face:. This means we don't need the ugly loading, just need to make this change in a few places.

Reply:
Yup! 👍